plugin/bookmark.vim

```diff
@@ -302,6 +307,11 @@ command! -nargs=? BookmarkMoveUp call s:move_relative(<q-args>, -1)
 command! -nargs=? BookmarkMoveDown call s:move_relative(<q-args>, 1)
 command! -nargs=? BookmarkMoveToLine call s:move_absolute(<q-args>)
 
+command! -bang -nargs=? -complete=buffer FzfBookmarks call fzf#vim#ag(<q-args>, {
+            \ 'source': fzf#bookmarks#list(),
+            \ 'sink': function('fzf#bookmarks#open'), 'down': '30%',
+            \ 'options': '--prompt "Bookmarks  >>>  "'})
```

Comment:
Thanks for this PR!
Do you how to make this create a preview window within the FZF split ? I've tried various combinations to no avail.

Reply:
I could be wrong, but I don't think it's possible to pass the selected line to a vim function. Fzf seems to pass it to the shell as `{}`, as in `--preview "echo {}"`.

Maybe passing it to a shell script (something like `--preview "sh vim_fzf_bookmarks.sh {}"`) that would parse the line, extract filename and line number, then use `sed` to show around those lines (like described [here](https://serverfault.com/questions/101900/read-specified-range-of-lines-from-a-file)).

Not sure where the shell script would go, though, or how to make it work under Windows (if it worked on Linux/Mac).

Reply:
ftr I've opened [a PR](https://github.com/mg979/vim-bookmarks/pull/1) on @mg979 's fork to deal with FZF previews.